### PR TITLE
Update command summary and add internship use cases

### LINKED ISSUE #110: Update UG command summary and exit command

### PR DESCRIPTION
**Pull request by ZoroarkDarkrai**
#64 #115

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -118,7 +118,66 @@ Use case ends.
...
-### Use case: UC05 - Add an internship application
+### Use case: UC05 - Add an internship
+
+#### MSS
+
+Precondition: User is on the Companies page <br/>
+Guarantees: Addition of internship is successful
+
+1.  User requests to add an internship to a company and provides the relevant details.
+2.  InternHunter adds the internship to the list of internships of the company.
```
Comment (orzymandias) [inline docs/DeveloperGuide.md]: I think you may be missing a line break here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -275,18 +275,21 @@ Format: `exit`
...
+&nbsp;          | **Edit**   | `edit com INDEX [n/COMPANY_NAME] [a/ADDRESS] [e/EMAIL] [p/PHONE_NUMBER] [t/INDUSTRY_TYPE]...`
+&nbsp;          | **View**   | `view com INDEX`
+**Internship**  | **Add**    | `add int INDEX j/JOB_TITLE [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
+&nbsp;          | **Delete** | `delete int INDEX`
+&nbsp;          | **Edit**   | `edit int INDEX i/INDEX [j/JOB_TITLE] [p/PERIOD] [w/WAGE] [r/REQUIREMENT]...`
+**Internship**  | **Add**    | `add app INDEX i/INDEX [s/STATUS] [d/STATUS_DATE]`
+**Application**`| **Delete** | `delete app INDEX`
+&nbsp;          | **Edit**   | `edit app INDEX [s/STATUS] [d/STATUS_DATE]`
+&nbsp;          | **View**   | `view app INDEX`
+**Profile**     | **Add**    | `add me c/CAT d/DESCRIPTORS`
```
Comment (orzymandias) [inline docs/UserGuide.md]: The profile command is updated with `[t/TITLE]` and `[d/DESCRIPTORS]...`

**Review by orzymandias — CHANGES_REQUESTED** [review]

**Review by orzymandias — APPROVED** [review]
LGTM, good work! Some merge conflicts to resolve first.

**Comment by ZoroarkDarkrai** [comment]
> LGTM, good work! Some merge conflicts to resolve first.

Fixed!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -138,6 +138,66 @@ Use case ends.
   Use case resumes from step 1.
 
 ### Use case: UC05 - Add an internship application
```
Comment (orzymandias) [inline docs/DeveloperGuide.md]: I think you have have missed out deleting this UC05

Comment (ZoroarkDarkrai) [reply docs/DeveloperGuide.md]: oh yeah, thanks for noticing :sweat_smile:

**Pull request merged**